actions/src/main/java/com/redhat/service/bridge/actions/webhook/WebhookAction.java

```diff
@@ -56,10 +58,11 @@ public ActionInvoker getActionInvoker(ProcessorDTO processor, BaseAction baseAct
         String endpoint = Optional.ofNullable(baseAction.getParameters().get(ENDPOINT_PARAM))
                 .orElseThrow(() -> buildNoEndpointException(processor));
         if (baseAction.getParameters().containsKey(USE_TECHNICAL_BEARER_TOKEN) && baseAction.getParameters().get(USE_TECHNICAL_BEARER_TOKEN).equals("true")) {
-            if (!webhookTechnicalBearerToken.isPresent()) {
-                throw new TechnicalBearerTokenNotConfiguredException("A webhook action needed the technical bearer token but it was not configured.");
+            Optional<AbstractOidcClient> abstractOidcClient = oidcClients.stream().filter(x -> x.getName().equals(OidcClientConstants.WEBHOOK_OIDC_CLIENT_NAME)).findFirst();
```

Comment:
We should probably name the bean so that we can explicitly select which bean to inject. Probably current code is still necessary because `actions` are injected also in the manager but not used (we need to refactor this)

Reply:
Yes, we need to refactor the `actions` module

Reply:
Can you please create a ticket? I think we should at refactor each action module to something like `action-sendtobridge-api`, `action-sendtobridge-validator` and `action-sendtobridge-runtime` (but maybe API and runtime are enough)